scripts/deploy.js

```diff
@@ -1,15 +1,56 @@
 const { Web3Connection, ERC20, NetworkFactory, Network } = require("@taikai/dappkit");
+const { exit } = require("process");
+
+
+const usage =  `------------------------------------------------------------------------- 
+  WebNetwork v1 Smart Contracts Deploy Script 🚀  
+-------------------------------------------------------------------------
+
+Usage: $0 -n <network> [-u <rpcUrl> -e <env file>] 
+
+<network> =  development | moonbase | kovan | mainnet | ropsten | seneca | afrodite | custom`
+
+const epilog = 'TAIKAI Copyright 2022'
+
+function showErrorAndHelp(message) {
+  return console.error(`
+${usage}
+
+💥 Failed: ${message}
+
+${epilog}
+`);
+}
 
 const argv = require('yargs')
     .alias('e', 'envFile')
     .describe('e', 'Environment file to load')
     .default('e', '.env')
+    .option('u', {
+      alias: "url",
+      describe: "EVM RPC URL",
+      type: 'string',
+      nargs: 1,
+    })  
     .alias('n', 'network')
     .describe('n', 'Ethereum Blockchain')
     .default('n', 'development')
+    .choices(["development", "moonbase", "kovan", "mainnet","ropsten", "seneca", "afrodite", "custom"])
+    .option('k', {
+      alias: "ownerKey",
+      describe: "Owner Private key",
+      type: 'string',
+      nargs: 1,
+    })  
+    .option('a', {
+      alias: "ownerAddress",
+      describe: "Owner Eth Address",
+      type: 'string',
+      nargs: 1,
+    })  
     .alias('h', 'help')
     .help('help')
-    .usage('Usage: $0 [options] <file>')
+    .usage(usage)
```

Comment:
I'm not sure, but I would think that if you don't provide this `yarn` will provide the `help` thing -- that way we don't have to write one by hand ;)